src/plone/exportimport/exporters/content.py

```diff
@@ -127,7 +132,7 @@ def dump_metadata(self) -> Path:
         filepath = self.base_path / "__metadata__.json"
         return self._dump(metadata, filepath)
 
-    def dump(self) -> List[Path]:
+    def dump(self, **kwargs) -> List[Path]:
```

Comment:
This doesn't appear to be used?

Reply:
Ah, right, this was a left-over from a different try.  I have removed it.